manifests/proxy.pp

```diff
@@ -347,6 +353,8 @@
   $hostname                        = $zabbix::params::proxy_hostname,
   $listenport                      = $zabbix::params::proxy_listenport,
   $sourceip                        = $zabbix::params::proxy_sourceip,
+  $enableremotecommands            = $zabbix::params::proxy_enableremotecommands,
```

Comment:
can you please add datatypes for both values? I assume both varaibles can only take integers with 0 as lowest value? So this could work:
```suggestion
   Integer[0] $enableremotecommands = $zabbix::params::proxy_enableremotecommands,
```